.github/workflows/ci.yml

```diff
@@ -94,6 +96,39 @@ jobs:
         env:
           RUSTFLAGS: '-D warnings'
 
+  cargo-test-workspace-all-features:
+    needs: cancel-previous-runs
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_PASSWORD: my-secret
+          POSTGRES_DB: postgres
+          POSTGRES_USER: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+      - run: cargo install diesel_cli
+      - run: |
+          cd schema
+          diesel migration run
+          cargo fmt
+      - uses: Swatinem/rust-cache@v1
```

Comment:
I think if we put this cache step right before the `cargo install diesel_cli` step, it might also cache diesel too.

Reply:
ok, I moved it up. Looks like it works though, tests are passing.

Reply:
Nice :+1: just need to add the new job to the list of `needs` under the `validation-complete` job now.